packages/slice-machine/components/App/index.tsx

```diff
@@ -31,6 +31,7 @@ const RemoveDarkMode: React.FunctionComponent = ({ children }) => {
 type AppProps = {
   theme: () => Theme;
   serverState: ServerState | undefined;
+  children: React.ReactElement;
```

Comment:
why we need this ? The type React.FunctionComponent already add the children props to the type